YSE_App/chime/chime_kko.py

```diff
@@ -36,31 +36,16 @@ def init_kko(clean:bool=True):
 
     return
 
-    # Load up the table
-    csv_file = os.path.join(
```

Comment:
Am assuming this is redundant code that has been moved elsewhere because the code that's replacing it is just measuring DMISM. Just checking.